packages/typespec-azure-resource-manager/lib/common-types/private-links.tsp

```diff
@@ -82,20 +120,9 @@ union PrivateEndpointServiceConnectionStatus {
  * The name of the private endpoint connection associated with the Azure resource.
  * @template Segment The resource type name for private endpoint connections (default is privateEndpointConnections)
  */
-@added(CommonTypes.Versions.v4)
```

Comment:
Do these exist in v3?  Why are we removing these decorators?

Reply:
Private link existed in v3